docs/source/code_examples/tutorials/ontologies/copying_ontology.py

```diff
@@ -0,0 +1,2 @@
+ontology = user_client.get_ontology('<ontology_hash>')
```

Comment:
If you look at the other examples, I think they all (at least new ones) include full imports and scripts that are (almost) ready to be copy, pasted, and used. 
This was a requirement from @eric-encord who asked for this so clients can use these examples quicker.

I do not have a strong opinion on whether we want to keep that pattern (good, because it's easy. Bad, because it repeats a lot of stuff) or make the examples leaner. But if we break that pattern it should be a very conscious decision.

Reply:
I think that makes sense, even at the cost of repetition we want people to be able to copy and paste things. 

Having looked at other examples, to my understanding only `from encord import EncordUserClient` is applicable here? Let me know if there's anything I've missed. 

These sort of basics are what I intend to clarify going forward and migrating to ReadMe, but for now just want to add this example to the existing documentation.

Reply:
So if you look at other examples you will probably see that we properly initialise the `user_client` as well.

I'm also going to say here that I highly recommend not moving away from sphinx to readme for python docs